pkgs/development/libraries/jbig2dec/default.nix

```diff
@@ -1,18 +1,18 @@
-{ stdenv, fetchurl, python3, autoconf }:
+{ stdenv, fetchurl, python3, autoreconfHook }:
 
 stdenv.mkDerivation rec {
-  name = "jbig2dec-0.17";
+  name = "jbig2dec-0.18";
```

Comment:
Can we split this into `pname` and `version`?

Reply:
Sure, updated (& rebased on current staging), thanks!